waypoint_loader_state_topic_name: /planning/final_waypoints

waypoint_loader_visual_topic_name: /map/global_path_rviz

node_rate: 1   # [Herz]
